### ISSUE TITLE
api/v2/apikeys -- 404

### ISSUE THREAD

**Issue by ScubaDrew** [drakkan/sftpgo#562]
Testing on [stoplight](https://sftpgo.stoplight.io/docs/sftpgo/openapi.yaml/paths/~1apikeys/get) and testing on my local install, I am getting 404 errors trying to use the `apikeys` endpoint:

![image](https://user-images.githubusercontent.com/18055237/135201603-a0bbd367-3cc3-474c-8362-fb05f3f69f3c.png)

```
sftpgo-01-c4bc79f8f-5sbh5 {"level":"info","time":"2021-09-29T04:04:59.015","sender":"httpd","method":"GET","proto":"HTTP/1.1","remote_addr":"127.0.0.1:58242","request_id":"sftpgo-01-c4bc79f8f-5sbh5/Ao9iZdPf9Q-000070","uri":"http://localhost:8080/api/v2/apikeys","user_agent":"PostmanRu
ntime/7.28.4","resp_status":404,"resp_size":24,"elapsed_ms":0}
```

Using a request like:

```
curl --location --request GET 'localhost:8080/api/v2/apikeys' \
--header 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWxxxI6MTYzMjg4ODk4MSwianRpIjoiYzU5dWE0OWduZnR0dDMwZm45YWciLCJuYmYiOjE2MzI4OxxxNpb25zIjpbIioiXSwic3ViIjoidXo0RkJhN1hBbTNiN2ZFV2FJRjRSZHFPWDI3VktzYkNQTFJDQzN1ZXVQQT0iLCJ1c2VybmFtZSI6Im51b3JkZXItc3RnMxxQ.liY89B5kvLJmyB9dGBGcKPK0lxxxz6oo3_4' \
--header 'Content-Type: application/json' \
--data-raw '{
  "name": "stg04-admin",
  "scope": 1,
  "description": "stg04-admin apikey",
  "admin": "stg04-admin"
}'
```

**Comment by drakkan** [comment]
Hi,

API keys were added after v2.1.x and are currently only in the development branch. They will be included in v2.2.x. If you need  API keys please use an edge based docker image or download a recent build artifact for your OS/distro.

If you want to use a released version please refer to the OpenAPI schema in the 2.1.x [branch](https://raw.githubusercontent.com/drakkan/sftpgo/2.1.x/httpd/schema/openapi.yaml)

**Comment by ScubaDrew** [comment]
Thanks, @drakkan ! Great stuff.
